examples/sessions/src/main.rs

```diff
@@ -79,28 +91,44 @@ where
             .await
             .expect("`MemoryStore` extension missing");
 
-        let headers = req.headers().expect("other extractor taken headers");
+        let session_cookie =
+            if let Ok(TypedHeader(cookie)) = TypedHeader::<Cookie>::from_request(req).await {
+                cookie.get(AXUM_SESSION_COOKIE_NAME).map(|x| x.to_owned())
+            } else {
+                None
+            };
```

Comment:
```suggestion
            Option::<TypedHeader<Cookie>>::from_request(req).await.and_then(
                |cookie| Some(cookie.get(AXUM_SESSION_COOKIE_NAME)?.to_owned())
            );
```

Reply:
Could also keep the `Option<TypedHeader>` in a local variable so `to_owned()` is not needed.